pvlib/test/test_forecast.py

```diff
@@ -75,6 +75,40 @@ def test_process_data(model):
                               .format(model, variable))
 
 
+@requires_siphon
+def test_bad_kwarg_get_data():
+    amodel = NAM()
```

Comment:
please add reference comment such as `# GH issue 745` so that future code readers will know where to look for more context.